externals/simplecpp/simplecpp.cpp

```diff
@@ -2763,7 +2763,7 @@ static std::string getIncludePathFileName(const std::string &includePath, const
 
 static std::string openHeaderIncludePath(std::ifstream &f, const simplecpp::DUI &dui, const std::string &header)
 {
-    for (std::list<std::string>::const_iterator it = dui.includePaths.begin(); it != dui.includePaths.end(); ++it) {
```

Comment:
simplecpp should be updated upstream

Reply:
Feel free to split that. I will not do that, as this separation of simplecpp from cppcheck has no real advantages and just makes life harder.